tensorboard_plugin_wit/wit_plugin.py

```diff
@@ -86,6 +86,24 @@ def __init__(self, context):
                             'authorized_groups' in context.flags and
                             context.flags.authorized_groups != '')
 
+    self.custom_predict_fn = None
+    if context.flags.custom_predict_fn:
+      try:
+        import importlib.util as iu
+        spec = iu.spec_from_file_location("custom_predict_fn", context.flags.custom_predict_fn)
+        module = iu.module_from_spec(spec)
+        spec.loader.exec_module(module)
+        self.custom_predict_fn = module.custom_predict_fn
+        logger.info("custom_predict_fn loaded.")
+
+      except Exception as e:
+        logger.info("Failed to load the custom predict function.")
+        logger.info(str(e))
+        # Show the error on the terminal.
+        print("Failed to load the custom predict function.")
```

Comment:
nit: Do we need this print if the above logger uses .error instead? I think that should propagate to terminal I think.

Slightly off topic from this PR, but @jameswex I also realized we import tb_logging and logging, but don't use tb_logging.get_logger, is there a reason we still import it?

Reply:
accidentally left that import in when we moved to standard logging. it can be removed

Reply:
also agree that the print statements can be removed assuming use of logging.error instead of .info.